### ISSUE TITLE
Checklist keyboard accessibility.

### ISSUE THREAD

**Issue by alan-cole** [vue-generators/vue-form-generator#477]
* **I'm submitting a ...**
  * [ x ] feature request

As tested on: https://jsfiddle.net/icebob/0mg1v81e/

* **What is the current behavior?**
- When keyboard tabbing through a form, focus will skip the checklist field. It is not able to be expanded / collapsed.

* **What is the expected behavior?**
- Focus is given to the checklist and on a keypress (Enter), the list can be expanded collapsed to access the fields within.

The current checklist is not accessible via keyboard. Changing the .info <div> to a <button> would solve this issue.

**Comment by dbaronov** [comment]
Yep, I can confirm that is the case, the VFG is not fully accessible. Can this be looked into, please?
